### DISCUSSION TITLE
Possible optimization in core of url_rewrites indexer

### DISCUSSION THREAD

**Discussion by seansan** [OpenMage/magento-lts#2098] [Ideas]
Possible optimization in core of url_rewrites indexer

https://github.com/magento/hackathon_january_2014/issues/42

**Comment by infabo** [comment]
There are several extension out in the wild, which alter/improve the url_rewrite indexing.

e.g. one of the well known: https://github.com/EcomDev/EcomDev_UrlRewrite

Has someone real-world production experience with either one and can give some 2cents?

**Comment by ghost** [comment]
The Dnd extension is good -- I have it installed. The main benefit was that I have so many non visible individually products that are members of grouped products. Without the patch, all those non-visible products have their own url.

**Comment by Flyingmana** [comment]
I know the EcomDev one is not maintained anymore and does not accept any PRs or similar.
But I would still prefer a fork of such a module over fixing it inside the core. We can talk again about it in ~2y

**Comment by dng-dev** [comment]
I would totally agree bringing a module which also used optimized sql queries and some features

**Comment by ProxiBlue** [comment]
https://github.com/EcomDev/EcomDev_UrlRewrite/issues/21

Caused endless pain with product URLs that Jade numerical endings. Abandoned it in the end as caused more issues than what it solved.
